python/tk_multi_reviewsubmission/renderer.py

```
@@ -164,14 +164,19 @@ def __create_output_node(self, path):
         #
         # These are either hard coded by a studio in a quicktime generation app
         # itself (like here) or part of the configuration - however there is
-        # often the need to have special rules to for example handle multi
+        # often the need to have special rules to, for example, handle multi
         # platform cases.
         #
 
         if sys.platform in ["darwin", "win32"]:
             # On the mac and windows, we use the quicktime codec
             node["file_type"].setValue("mov")
-            node["codec"].setValue("jpeg")
+            # Nuke 9.0v1 changed the codec knob name and added an encoder knob
+            # (which defaults to the new mov64 encoder/decoder) 
+            if not node.knob["codec"].setValue("jpeg"):
```

oops. misunderstood how Nuke handled nonexistent knob names. `node["codec"]` returns a `Obsolete_Knob` object which was silently hiding my idiocy.

Agreed with your comment about equality before inequality. update pending...